### PR TITLE
[BACKLOG-37966] Toolbar improvements for PIR+PDD

### PR DESCRIPTION
**Pull request by AAravind27**
@pentaho/wcag Please review, for toolbars that are used in PIR as well as PDD.

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 41s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl impl
```
---
:ok_hand: All tests passed!
 
Tests run: **40**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/mql-editor/job/PR-73/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: impl/src/main/resources/org/pentaho/commons/metadata/mqleditor/editor/xul/mainFrame.xul [hunk 1]
```diff
@@ -124,16 +124,16 @@
         </tree>
 
 
-        <hbox id="conditionsButtonContainer" width="575">
+        <toolbar id="conditionsButtonContainer">
           <label value="${conditions}" flex="1" />
-          <spacer id="conditionsSpacer" flex="1" />
-          <button id="conditionDown" disabled="true"
+          <toolbarspacer id="conditionsSpacer" flex="1" />
+          <toolbarbutton id="conditionDown" disabled="true"
```
Comment (dcleao) [inline impl/src/main/resources/org/pentaho/commons/metadata/mqleditor/editor/xul/mainFrame.xul:130]: Are all the attributes supported by `button` also supported by `toolbarbutton`?

**Review by dcleao — CHANGES_REQUESTED** [review]

File: impl/src/main/resources/org/pentaho/commons/metadata/mqleditor/editor/xul/mainFrame.xul (hunk 1 above)
Comment (AAravind27) [reply impl/src/main/resources/org/pentaho/commons/metadata/mqleditor/editor/xul/mainFrame.xul:130]: https://github.com/pentaho/pentaho-platform-plugin-dashboards/pull/570/files#r1341710831

Comment (dcleao) [reply impl/src/main/resources/org/pentaho/commons/metadata/mqleditor/editor/xul/mainFrame.xul:130]: I take your answer to mean: "As answered elsewhere, no, not all attributes are supported. However, the only unsupported attribute, `pen:disabledimage`, is not used in these instances.".

**Review by dcleao — APPROVED** [review]

File: impl/src/main/resources/org/pentaho/commons/metadata/mqleditor/editor/xul/mainFrame.xul (hunk 1 above)
Comment (AAravind27) [reply impl/src/main/resources/org/pentaho/commons/metadata/mqleditor/editor/xul/mainFrame.xul:130]: Yes, I should have phrased it like that.

**Review by hv-vamsi — APPROVED** [review]

**Pull request merged**